Lib/html/parser.py

```diff
@@ -46,7 +46,7 @@
           |"[^"]*"                   # LIT-enclosed value
           |(?!['"])[^>\s]*           # bare value
          )
-         (?:\s*,)*                   # possibly followed by a comma
+        \s*                          # handling spaces only
```

Comment:
```suggestion
        \s*                          # possibly followed by a space
```
Just one last nit, the previous comment made sense while going from handling both commas and spaces to spaces only, but by itself it is somewhat confusing, so I suggest to replace it with this.